keps/2076-kueuectl/README.md

```diff
@@ -339,7 +340,42 @@ so that the users won't have to remember to switch the command to kubectl.
 * `describe workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
 * `edit workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
 * `patch workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
-* `delete workload|wl|clusterqueue|cq|localqueue|lq|resourceflavor|rf`
+* `delete clusterqueue|cq|localqueue|lq|resourceflavor|rf`
+
+### Delete Workload
+
+Motivation:
+The pass-through delete workload command doesn't actually delete the Workload but 
+instead requeues it. This can confuse users, as the Workload remains listed even 
+after they attempt to delete it, due to its recreation.
+
+This command will delete the Workload and its corresponding Job(s). If the Workload 
```

Comment:
I think we should better define what "corresponding Job(s)" are. 
And maybe think about:
- how we ca address what I mentioned in https://github.com/kubernetes-sigs/kueue/pull/2992#discussion_r1765051596
- maybe mention the the effect of deleting parent object on workload sibling objects

Reply:
> how we ca address what I mentioned in https://github.com/kubernetes-sigs/kueue/pull/2992#discussion_r1765051596

We just looked at this case with @mimowo and found that the Deployment recreates the Pod and it will have a different name.

Reply:
I don't understand. The Deployment is just an example.

Reply:
The Workload associated with pod and we delete this Pod. So it's ok here. But I also need to check [FluxMiniclusters](https://kueue.sigs.k8s.io/docs/tasks/run/flux_miniclusters/).

Reply:
A user may want to delete the whole `Deployement`, but for that, we need to have a different workload.

Reply:
Even if the names are slightly different in case of replicasets/deployments, you end up deleting two objects and other two (having the same purpose) are created.

Reply:
I think this is ok for now, as the workload requested to be deleted by the user is gone, which was the main confusing factor in the original issue.

The support for Deployments is in initial phase.

Long term, we discuss introducing a concept of a workload fragment. Then workloads for Pods will be considered workload fragments, and we may display a parent workload which aggregates the fragments. Then deleting the parent workload would delete the Deployment instance, but I believe this work can be deferred for later, but cc @mwielgus in case we want to prioritize it more as part of the fix.

Reply:
> But I also need to check [FluxMiniclusters](https://kueue.sigs.k8s.io/docs/tasks/run/flux_miniclusters/).

Just to confirm, the situation in `FluxMiniclusters` is the same. It removes the Job associated with a Workload but then recreates both a new Job and a new Workload.

Reply:
> Just to confirm, the situation in `FluxMiniclusters` is the same. It removes the Job associated with a Workload but then recreates both a new Job and a new Workload.

With the same name ore a slightly different one?

Reply:
With different one.